cudamapper/src/overlapper.cpp

```diff
@@ -350,7 +351,7 @@ void Overlapper::rescue_overlap_ends(std::vector<Overlap>& overlaps,
 
         while (rescue_rounds < max_rescue_rounds)
         {
-            details::overlapper::extend_overlap_by_sequence_similarity(overlap, query_view, target_view, 100, 0.9);
+            details::overlapper::extend_overlap_by_sequence_similarity(overlap, query_view, target_view, extension, required_similarity);
```

Comment:
@edawson is this the change you were referring to?

Reply:
Yes, that's the one I wanted to make.